src/default_panels/StyleTracesPanel.js

```diff
@@ -78,6 +78,17 @@ const StyleTracesPanel = (props, {localize: _}) => (
       <Numeric label={_('Max Tube segments')} attr="maxdisplayed" />
     </PlotlySection>
     <MultiColorPicker label={_('Color')} attr="color" />
+    <TraceTypeSection name={_('Waterfall')} traceTypes={['waterfall']} mode="trace">
+      <LayoutSection attr="name">
+        <Radio
+          label={_('Mode')}
+          attr="waterfallmode"
+          options={[{label: _('Group'), value: 'group'}, {label: _('Overal'), value: 'overlay'}]}
```

Comment:
lol: https://plotly.github.io/schema-viewer/?attribute=waterfallmode
Label: 'Overlay' : )

Reply:
wow, how did that get in there!